apps/ledger-live-mobile/src/components/WebPlatformPlayer/index.js

```diff
@@ -122,20 +124,23 @@ const WebPlatformPlayer = ({ manifest, inputs }: Props) => {
 
     if (inputs) {
       for (const key in inputs) {
-        if (Object.prototype.hasOwnProperty.call(inputs, key)) {
+        if (
+          Object.prototype.hasOwnProperty.call(inputs, key) &&
+          inputs[key] !== undefined
+        ) {
           url.searchParams.set(key, inputs[key]);
         }
       }
     }
 
-    url.searchParams.set("backgroundColor", new Color(theme.colors.card).hex());
-    url.searchParams.set("textColor", new Color(theme.colors.text).hex());
+    url.searchParams.set("lang", locale);
+    url.searchParams.set("theme", theme.theme);
```

Comment:
`theme` is already provided by the `PlatformApp` component in the `inputs`.

Also we should keep `backgroundColor` & `textColor` as some live-apps might depend on it and we still provide them in LLD.